File: VBHotkeyWin/Form1.vb
Public Class Form1

    Private ghk As VBHotkeys.GlobalHotkey

    Private Sub Form1_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        ghk = New VBHotkeys.GlobalHotkey(VBHotkeys.Constants.ALT, Keys.O, Me)
        WriteLine("Trying to register ALT+O")
        If ghk.Register() Then
            WriteLine("Hotkey Registered")
        Else
            WriteLine("Hotkey failed to register!")
        End If
    End Sub

    Private Sub HandleHotkey()
        WriteLine("Hotkey pressed!")
    End Sub

    Protected Overrides Sub WndProc(ByRef m As System.Windows.Forms.Message)
        If (m.Msg = VBHotkeys.Constants.WM_HOTKEY_MSG_ID) Then
            HandleHotkey()
        End If
        MyBase.WndProc(m)
    End Sub

    Private Sub Form1_FormClosing(ByVal sender As System.Object, ByVal e As System.Windows.Forms.FormClosingEventArgs) Handles MyBase.FormClosing
        If Not ghk.Unregister() Then
            MessageBox.Show("Hotkey failed to unregister!")
        End If
    End Sub

    Private Sub WriteLine(ByVal message As String)
        TextBox1.Text &= message & Environment.NewLine
    End Sub
End Class

File: HotkeyWin/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using Hotkeys;

namespace HotkeyWin
{
    public partial class Form1 : Form
    {
        private Hotkeys.GlobalHotkey ghk;

        public Form1()
        {
            InitializeComponent();
            ghk = new Hotkeys.GlobalHotkey(Constants.ALT + Constants.SHIFT, Keys.O, this);
        }

        private void HandleHotkey()
        {
            WriteLine("Hotkey pressed!");
        }

        protected override void WndProc(ref Message m)
        {
            if (m.Msg == Hotkeys.Constants.WM_HOTKEY_MSG_ID)
                HandleHotkey();
            base.WndProc(ref m);
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            WriteLine("Trying to register SHIFT+ALT+O");
            if (ghk.Register())
                WriteLine("Hotkey registered.");
            else
                WriteLine("Hotkey failed to register");
        }

        private void Form1_FormClosing(object sender, FormClosingEventArgs e)
        {
            if (!ghk.Unregiser())
                MessageBox.Show("Hotkey failed to unregister!");
        }

        private void WriteLine(string text)
        {
            textBox1.Text += text + Environment.NewLine;
        }
    }
}
